addons/markers/functions/fnc_loadBriefingMarkers.sqf

```diff
@@ -0,0 +1,49 @@
+/*
+ * Author: SzwedzikPL
+ * Loads all post briefing markers and saves them as unique to assigned map
+ * Triggered by server on each client 0.5 sec after mission start
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+if (!hasInterface || {!GVAR(enableUniqueMaps)}) exitWith {};
+
+private _assignedMap = ACE_player call FUNC(getAssignedMap);
+if (_assignedMap == "") exitWith {};
+
+//can't update markers data if map is not unique
+private _mapID = _assignedMap call FUNC(getMapID);
+if (_mapID <= 0) exitWith {};
```

Comment:
If map classname is used as _mapId then this would become something like `if (_mapId == Base_Map)`